README.md

```diff
@@ -0,0 +1,36 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard
+
+This is an app that displays the scores on a leaderboard.
+
+![screenshot](./screenshot.PNG)
+
+This project is a to-do-list app.
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- webpack
+
+## Getting Started
+To get the content of this project locally you need to run this command in your terminal:
+- git clone your https://github.com/matovu-farid/leaderboard.git
+- cd project leaderboard
```

Comment:
Please add complete instructions to set the project up locally by adding `npm install` and `npm start`. This way a non-technical user is able to use the project.